.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const path = require("path");
+const rulesDirPlugin = require("eslint-plugin-rulesdir");
+
+rulesDirPlugin.RULES_DIR = path.join(__dirname, "lib/internal-rules");
+
+module.exports = {
+    root: true,
+    plugins: [
+        "eslint-plugin",
+        "node",
+        "rulesdir"
+    ],
+    extends: [
+        "./packages/eslint-config-eslint/default.yml",
+        "plugin:node/recommended",
+        "plugin:eslint-plugin/recommended"
+    ],
+    rules: {
+        "eslint-plugin/consistent-output": "error",
+        "eslint-plugin/no-deprecated-context-methods": "error",
+        "eslint-plugin/no-identical-tests": "error",
+        "eslint-plugin/prefer-output-null": "error",
+        "eslint-plugin/prefer-placeholders": "error",
+        "eslint-plugin/report-message-format": ["error", "[^a-z].*\\.$"],
+        "eslint-plugin/test-case-property-ordering": "error",
+        "eslint-plugin/test-case-shorthand-strings": "error"
```

Comment:
"eslint-plugin/no-identical-tests" has been added in `recommended`.

Reply:
Good catch, but I think we should remove it in a separate PR -- this PR is only supposed to change the format of the config file without modifying how it works.

Reply:
it has been removed in the master. so I think better not to add it again.

Reply:
Oh, oops. I hadn't realized that it was removed on `master`.